### PR TITLE
Kamu UI 293 data access panel integration with graphql api

### PR DESCRIPTION
**Pull request by dmitriy-borzenko**

### CODE REVIEW

File: resources/schema.graphql [hunk 1]
```diff
@@ -8,30 +8,30 @@
...
-  accountType: AccountType!
-  """
-  Avatar URL
-  """
-  avatarUrl: String
-  """
-  Indicates the administrator status
-  """
-  isAdmin: Boolean!
+	"""
```
Comment (zaychenko-sergei) [inline resources/schema.graphql:11]: Plz check formatting

File: src/app/api/kamu.graphql.interface.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // THIS FILE IS GENERATED, DO NOT EDIT!
-import { gql } from "@apollo/client/core";
+import { gql } from "apollo-angular";
```
Comment (zaychenko-sergei) [inline src/app/api/kamu.graphql.interface.ts:2]: Oops

File: resources/schema.graphql (hunk 1 above)
Comment (dmitriy-borzenko) [reply resources/schema.graphql]: Fixed

File: src/app/api/kamu.graphql.interface.ts (hunk 2 above)
Comment (dmitriy-borzenko) [reply src/app/api/kamu.graphql.interface.ts]: Fixed

**Pull request merged**